app/src/app/components/contents/packets/PacketHeader.tsx

```diff
@@ -1,13 +1,27 @@
 interface PacketHeaderProps {
+  displayName: string;
   packetName: string;
   packetId: string;
 }
 
-export default function PacketHeader({ packetName, packetId }: PacketHeaderProps) {
-  return (
-    <div>
-      <h2 className="text-2xl font-bold tracking-tight">{packetName}</h2>
-      <p className="text-muted-foreground">{packetId}</p>
-    </div>
-  );
-}
+export default function PacketHeader({ displayName, packetName, packetId }: PacketHeaderProps) {
```

Comment:
possoble to make displayName to allow undefined as well... saves you having to do displayName ?? "" everywhere.. and the check can just be if displayName

Reply:
Fair point. I'll still keep the verbose check in this file though because we need to see if the display name matches the name.